fe/fe-core/src/main/java/org/apache/doris/nereids/rules/analysis/BindRelation.java

```diff
@@ -139,9 +139,14 @@ private LogicalPlan bindWithCurrentDb(CascadesContext cascadesContext, UnboundRe
         }
         String catalogName = cascadesContext.getConnectContext().getCurrentCatalog().getName();
         String dbName = cascadesContext.getConnectContext().getDatabase();
-        TableIf table = cascadesContext.getTables() != null
-                ? cascadesContext.getTableByName(tableName)
-                : getTable(catalogName, dbName, tableName, cascadesContext.getConnectContext().getEnv());
+        TableIf table = null;
```

Comment:
already fixed by another PR: #19695 , please rebase after it be merged

Reply:
Changed the code to be identical with the other pr, so there should be no confliction.